scrapers/Cargo.toml

```diff
@@ -7,8 +7,12 @@ edition = "2021"
 
 [dependencies]
 db = { path = "../db" }
-csv = "1.3.0"
-reqwest = { version = "0.11.18" }
-thirtyfour = "0.31.0"
-tokio = { version = "1.30.0", features = ["full"] }
-sqlx = { version = "0.7", features = ["postgres", "macros"] }
+csv = "1"
+project-root = "0"
+regex = "1"
+reqwest = "0"
+scraper = "0"
+slugify = "0"
+sqlx = { version = "0", features = ["postgres", "macros"] }
+thirtyfour = "0"
+tokio = { version = "1", features = ["full"] }
```

Comment:
thanks for simplifying these dependencies I'll use this convention across the rest of the repo in a follow up